### ISSUE TITLE
FileNotFoundException on first use of 0.7.0

### ISSUE THREAD

**Issue by LuccoJ** [cyb3rko/pincredible#28]
Release from F-Droid running on Android 13.

Steps to reproduce (update: it seems to occur upon hitting Back after deleting the last entered PIN, likely because Back tries to send you right there, instead of leaving the app, which is possiblt not the most desireable behavior in the first place):

1. Open the app for the first time after installing it
2. Generate cells, input a new PIN, genete numbers
3. Save it with a name
4. Delete the entry you just saved
5. Hit Back

After this, I get the following error:

```
java.io.FileNotFoundException: /data/user/0/com.cyb3rko.pincredible/files/p391c30541690b2a65a2d63485: open failed: ENOENT (No such file or directory)
	at libcore.io.IoBridge.open(IoBridge.java:574)
	at java.io.FileInputStream.<init>(FileInputStream.java:160)
	at mh.c(Unknown Source:2)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment.a0(SourceFile:28)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment.c0(SourceFile:71)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment$onViewCreated$1.h(SourceFile:31)
	at s9.b(SourceFile:9)
	at e8.r1(SourceFile:179)
	at e8.R1(SourceFile:12)
	at e8.H0(SourceFile:129)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment.M(SourceFile:118)
	at androidx.fragment.app.a.f(SourceFile:255)
	at androidx.fragment.app.a.k(SourceFile:146)
	at hr.A(SourceFile:134)
	at hr.W(SourceFile:92)
	at hr.y(SourceFile:74)
	at ui.run(SourceFile:37)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7918)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
	Suppressed: nj: [de0{Cancelling}@f0a919f, Dispatchers.Main.immediate]
Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
	at libcore.io.Linux.open(Native Method)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:563)
	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:274)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:563)
	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7804)
	at libcore.io.IoBridge.open(IoBridge.java:560)
	... 24 more
```

**Comment by cyb3rko** [comment]
Can confirm this issue, thank you for your report.
Will be investigated.

**Comment by cyb3rko** [comment]
Whoops, closed by mistake.

**Comment by cyb3rko** [comment]
Should be corrected in [Version 0.7.1](https://github.com/cyb3rko/pincredible/releases/tag/v0.7.1).  
Can you double check?
